statusboard/src/components/PageContents/PageContents.css

```diff
@@ -1,5 +1,5 @@
 .contents {
-  display: block;
+  flex: 1;
```

Comment:
This looks like it's not affecting anything. Never been great with `flex-grow`, etc., so you can do what you want with it.

Reply:
nvm - when it's not tall enough to take up vh, it needs this.